bin/reth/src/prometheus_exporter.rs

```diff
@@ -0,0 +1,13 @@
+//! Prometheus exporter
+
+use metrics_exporter_prometheus::PrometheusBuilder;
+use metrics_util::layers::{PrefixLayer, Stack};
+
+pub(crate) fn initialize_prometheus_exporter() {
```

Comment:
instead of panicking here, let's add a `try_` option as well, we should have eyre as error handling here so letting this return a result instead is nicer.